### PR TITLE
Fix: NFT image URL request changed to LCD

### PR DESCRIPTION
**Pull request by faisalnaveedRNS**
## Description
NFT Image URL issue was fixed, Image now get directly from LCD end Point
Fixes #
-   Dead Code Removed
-   Axios request was added
-  Component was directly rendered 

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Run `meteor npm run lint`
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: imports/api/nfts/server/methods.js [hunk 1]
```diff
@@ -1,111 +1,149 @@
...
+                "?action=resell_nft&recipe_id=" +
+                item.recipeID +
+                "&cookbook_id=" +
+                nft.cookbookID +
+                "&nft_amount=1";
+              item.resalelink = resalelink;
+
+              bulkNfts.find({ ID: item.ID }).upsert().updateOne({ $set: item });
+            } catch (e) {
+              //bulkNfts.find({ ID: trade.itemOutputs[0].itemID }).upsert().updateOne({ $set: nft });
```
Comment (mustafapylons) [inline imports/api/nfts/server/methods.js]: this dead code is it necessary for testing/demo?

File: imports/api/recipes/server/methods.js [hunk 2]
```diff
@@ -1,150 +1,179 @@
...
+
+        if (Meteor.isServer) {
+          loading = false;
+          transactionsExist = !!transactions;
+        } else {
+          transactionsExist = !loading && !!transactions;
+        }
+      }
+
+      // if(!transactionsExist){
```
Comment (mustafapylons) [inline imports/api/recipes/server/methods.js]: same here, is there any purpose for this commented code?

File: imports/api/recipes/server/methods.js [hunk 3]
```diff
@@ -1,150 +1,179 @@
...
-                }
-                else{
-                    transactionsExist = !loading && !!transactions;
-                }
+          } catch (e) {
+            console.log(e);
+          }
+          recipe.cookbook_owner = cookbook_owner;
+          recipe.creator = creator;
+          // if(recipe.entries != undefined && recipe.entries.itemOutputs.length > 0 && recipe.entries.itemOutputs[0].strings != undefined){
```
Comment (mustafapylons) [inline imports/api/recipes/server/methods.js]: same here for the dead code.

**Review by mustafapylons — COMMENTED** [review]
Couple of places with dead code, unless they are for testing/demo purpose, it can be removed.
Also it might be better to sanitize all the inputs eventually in the code.
Otherwise clean code.

File: imports/api/nfts/server/methods.js (hunk 1 above)
Comment (faisalnaveedRNS) [reply imports/api/nfts/server/methods.js]: I don't think so we need it later on

Comment (faisalnaveedRNS) [reply imports/api/nfts/server/methods.js]: we will aslo remove this dead code

File: imports/api/recipes/server/methods.js (hunk 2 above)
Comment (faisalnaveedRNS) [reply imports/api/recipes/server/methods.js]: will remove this dead code

File: imports/ui/home/Home.jsx [hunk 4]
```diff
@@ -1,61 +1,78 @@
...
+      querys["recipe_id"] != null &&
+      querys["nft_amount"] == 1 &&
+      querys["cookbook_id"] != null
+    ) {
+      this.setState({
+        recipeExist: true,
+        recipe_id: querys["recipe_id"],
+        cookbook_id: querys["cookbook_id"],
+      });
+      console.log("querys['recipe_id']", querys["recipe_id"]);
```
Comment (mustafapylons) [inline imports/ui/home/Home.jsx:36]: is this logging here necessary?

File: imports/ui/easel_buy/EaselBuyContainer.js [hunk 5]
```diff
@@ -1,77 +1,76 @@
...
+  }
 
-    if (Meteor.isServer || !loading) { 
-        selectedRecipe = Recipes.findOne({ ID: recipe_id });
+  if (selectedRecipe != null) {
+    name = selectedRecipe.name;
+    description = selectedRecipe.description;
+    const coinInputs = selectedRecipe.coinInputs;
+    if (coinInputs.length > 0) {
+      if (coinInputs[0].coins[0].denom == "USD") {
```
Comment (mustafapylons) [inline imports/ui/easel_buy/EaselBuyContainer.js:34]: "USD" can be in a constant.

File: imports/api/nfts/server/methods.js [hunk 6]
```diff
@@ -1,111 +1,139 @@
...
+                date.getFullYear() * 1000 * 360 * 24 * 30 * 12 +
+                date.getMonth() * 1000 * 360 * 24 * 30 +
+                date.getDay() * 1000 * 360 * 24 +
+                date.getHours() * 1000 * 360 +
+                date.getMinutes() * 1000 * 60 +
+                date.getSeconds() * 1000 +
+                date.getMilliseconds();
+              item.tradeable = true;
+
+              let resalelink =
```
Comment (mustafapylons) [inline imports/api/nfts/server/methods.js:94]: we have to sanitize the links.

File: imports/api/recipes/server/methods.js [hunk 7]
```diff
@@ -1,150 +1,133 @@
...
+          .map((p) => p.ID)
+      );
+
+      let recipeIds = [];
+      if (recipes.length > 0) {
+        const bulkRecipes = Recipes.rawCollection().initializeUnorderedBulkOp();
+
+        for (let i in recipes) {
+          let recipe = recipes[i];
+          let deeplink =
```
Comment (mustafapylons) [inline imports/api/recipes/server/methods.js:56]: should sanitize links.

**Review by mustafapylons — COMMENTED** [review]
Couple issues, otherwise good code.

File: imports/api/recipes/server/methods.js (hunk 2 above)
Comment (BlackPanthar) [reply imports/api/recipes/server/methods.js]: Reviewed

File: imports/api/recipes/server/methods.js (hunk 3 above)
Comment (BlackPanthar) [reply imports/api/recipes/server/methods.js]: Reviewed

File: imports/ui/easel_buy/EaselBuyContainer.js (hunk 5 above)
Comment (BlackPanthar) [reply imports/ui/easel_buy/EaselBuyContainer.js:34]: Reviewed

File: imports/api/recipes/server/methods.js (hunk 7 above)
Comment (BlackPanthar) [reply imports/api/recipes/server/methods.js:56]: Reviewed

**Review by BlackPanthar — APPROVED** [review]
Feedback given should be used in subsequent PRs.

**Pull request merged**